src/content/app/genome-browser/components/drawer/components/sequence-view/DrawerSequenceView.tsx

```diff
@@ -38,6 +44,7 @@ const sequenceLabelsMap: Record<SequenceType, string> = {
 
 // TODO: we probably also want to pass a sequence header in order to be able to blast it
 type Props = {
+  genomeId: string;
```

Comment:
we dont seems to be using this anywhere in the component? why has this been added in the props?